content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -424,3 +427,21 @@ namespace.
 
 When each team has their own mesh, cross-mesh communication follows the
 concepts described in the [multiple meshes](#multiple-meshes) model.
+
+## Mesh expansion
+
+In same cases you might have to deploy some of the services on virtual machines or bare metal computers. The
+reasons for deployments on VMs/bare metal computers could be special hardware requirements, tighter isolation, tighter integration with the host's OS, legacy deployments.
+
+Having some of the services on VMs/bare metal, you may want to connect the services into a mesh and benefit from all the
+service mesh features: traffic management, security, observability, and others. You can deploy the Istio control plane(s)
+on some of your clusters and expand the mesh of the cluster(s) by adding VMs/bare metal computers into the mesh.
```

Comment:
Again, I'd lose the phrase "expand the mesh". Let's go with Frank's phrasing of "integrate with...".

Reply:
Addressed in https://github.com/istio/istio.io/pull/6209/commits/26fc6d5dcbe19d51ecd80c46190beedf0aec6456.